amplipi/streams.py

```diff
@@ -144,7 +156,7 @@ def restart(self):
       self.send_cmd('stop')
     except:
       logger.info(f'Stream {self.name} does not have a stop response')
-    last_src = self.src # Disconnect sets self.src to none, so temp variable used to keep track
+    last_src = self.src  # Disconnect sets self.src to none, so temp variable used to keep track
```

Comment:
Whats going on with the formatter here? I'm seeing a fair amount of lines that are just spacing changes.

Reply:
Looks like you are using vscode's autoformatting. We should probably do this to our entire codebase. PR made for this here: https://github.com/micro-nova/AmpliPi/pull/773